### PR TITLE
E_CLASSROOM-251 [FE/BE] Dynamic data for questions/choices

### PR DESCRIPTION
**Pull request by CaneteJoash**
### Issue Link
https://framgiaph.backlog.com/view/E_CLASSROOM-251
### Definition of Done
- [x] Dynamic data is displayed
- [x] Answers with the corresponding icons are displayed

### Commands to Run
N/A

### Notes
- Try visit different id like 
- http://localhost:3003/admin/quizzes/3
- http://localhost:3003/admin/quizzes/4

#### Main note
- N/A
#### Pages Affected
- http://localhost:3003/admin/quizzes/1

### Scenarios/ Test Cases
- [x] Dynamic data is displayed
- [x] Answers with the corresponding icons are displayed
#### User Interface
N/A
#### User Experience 
N/A
#### Functionality
N/A
### Screenshots
image
image

### CODE REVIEW

**Comment by Teraaay** [comment]
### Working
- [x] Dynamic data is displayed
- [x]  Answers with the corresponding icons are displayed

### Comment/s
- [x] Please link the quizzes to their corresponding quiz overview.
>  In the Quiz List page, when clicking the edit button of a specific Quiz it should redirect to its corresponding Quiz Overview.
- [x] Redirect back to the Quiz List page when clicking the back button

### Screenshot/s | Screenrecord/s
> DATA OVERVIEW DYNAMIC DATA

------
- Total Test Time: 0.25 hr

**Comment by Teraaay** [comment]
### Approved PR #104
- [x] Dynamic data is displayed
- [x] Answers with the corresponding icons are displayed
- [x] Able to link quiz to its corresponding quiz overview
- [x] Able to link back button back to the quiz list page

### Screenshot/s
> QUIZ OVERVIEW

----
- Total Test time: 0.5 hr
- Total Iterations: 1
- Total Comments: 2

File: src/pages/admin/quizzes/QuizDetail/index.module.scss [hunk 1]
```diff
@@ -41,9 +41,10 @@
 .backButton {
   cursor: pointer;
   font-size: 27px;
+  color: #48535B;
```
Comment (karlolee50) [inline src/pages/admin/quizzes/QuizDetail/index.module.scss]: use global variable

**Review by karlolee50 — CHANGES_REQUESTED** [review]
Review time: 10 minutes
Comment/s: 1

**Review by karlolee50 — APPROVED** [review]

**Pull request merged**